zkevm-circuits/Cargo.toml

```diff
@@ -16,7 +16,7 @@ sha3 = "0.10"
 digest = "0.7.6"
 array-init = "2.0.0"
 paste = "1.0"
-bus-mapping = { path = "../bus-mapping" }
+bus-mapping = { path = "../bus-mapping", features = ["test"] }
```

Comment:
It is possible to make zkevm circuits only need busmapping test feature when zkevm circuit is also in test mode？

Reply:
Yes I will make the change